### ISSUE TITLE
Get "Missing path for binary" panic when using `[[bin]]` without `path` field

### ISSUE THREAD

**Issue by chris13524** [LukeMathWalker/cargo-chef#14]
Panic is triggered [here](https://github.com/LukeMathWalker/cargo-chef/blob/main/src/skeleton.rs#L108). According to the [cargo docs](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#the-path-field), when the `path` field isn't included it is assumed to be the [inferred path](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#target-auto-discovery). I think this tool should follow the same spec.

This could perhaps be a broader issue with cargo-manifest and the way [`autoset()`](https://github.com/LukeMathWalker/cargo-manifest/blob/master/src/lib.rs#L200) works.

**Comment by LukeMathWalker** [comment]
It should be fixed now in master :ok_hand:

**Comment by LukeMathWalker** [comment]
The fix has been released in `cargo-chef:0.1.7`.

**Comment by thrykol** [comment]
Looks like there may have been a regression at some point.  Using `cargo-chef:0.1.34` with `rust:1.59.0-alpine3.5` image, I'm seeing the issue again.  Adding `path = "src/bin/mybin/main.rs"` required for the `chef cook` layer to succeed.

Here is the `bin` section from my `Cargo.toml`
```
[[bin]]
name = "server"
path = "src/bin/server/main.rs"
required-features = ["axum", "tokio"]```
